etc/profile-a-l/joplin.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for joplin
+# Description: A note taking and to-do application
+# This file is overwritten after every install/update
+quiet
+
```

Comment:
```suggestion
```

Reply:
this was added to avoid following output. 
```
...
Reading profile /etc/firejail/allow-bin-sh.inc
Reading profile /etc/firejail/allow-nodejs.inc
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-exec.in
...
```

Reply:
I forget that we it's a cli program. Right. Leave the `quiet`, but remove the blank line.